catalog/app/containers/Profile/constants.js

```diff
@@ -11,8 +13,29 @@ export const UPDATE_PLAN = 'app/Profile/UPDATE_PLAN';
 export const UPDATE_PLAN_ERROR = 'app/Profile/UPDATE_PLAN_ERROR';
 export const UPDATE_PLAN_SUCCESS = 'app/Profile/UPDATE_PLAN_SUCCESS';
 
-export const PLANS = {
+// public cloud plans
+export const PLANS = config.team ? {
+  // team plans
+  team_unpaid: {
+    confirmTitle: 'Cancel team account?',
+    confirmBody: 'Your team will lose access to all data.',
+    cost: 0,
+    menu: 'Team (30-day trial)',
+    rank: 0,
+    statusIcon: 'warning',
+    statusMessage: 'Temporary Service plan. Upgrade to avoid data loss.',
```

Comment:
I don't think we should use words like "data loss"... it may result in "customer loss".